### PR TITLE
Fix: Timeout issue

### PR DESCRIPTION
**Pull request by robbiedood**
Thanks for @StefanFabian request.
The PR is to resolve tf2_ros's buffer threading error and slash '/' compatibility as described in #23 .

### CODE REVIEW

File: hector_exploration_node/src/exploration_node.cpp [hunk 1]
```diff
@@ -39,7 +39,7 @@ class SimpleExplorationPlanner
   SimpleExplorationPlanner()
   {
     ros::NodeHandle nh;
-
+    buffer_.setUsingDedicatedThread(true)
```
Comment (StefanFabian) [inline hector_exploration_node/src/exploration_node.cpp]: This is not going to compile;

Comment (lukelu520) [reply hector_exploration_node/src/exploration_node.cpp]: Thanks for the catch, the semicolon typo has been fixed.

**Comment by StefanFabian** [comment]
Could you verify that the changes I made compile and work for you?

**Comment by robbiedood** [comment]
@StefanFabian , the changes you made are nice; compiled successfully and it's working very well !

**Pull request merged**